core/Dockerfile.template

```diff
@@ -11,13 +11,13 @@ COPY package*.json ./
 
 RUN npm ci
 
-FROM balenalib/%%BALENA_ARCH%%-alpine-node:12.19.1-3.12-run-20201211
+FROM balenalib/%%BALENA_ARCH%%-alpine:3.12-run-20211030
 
 ENV UDEV 1
 ENV DBUS_SYSTEM_BUS_ADDRESS unix:path=/host/run/dbus/system_bus_socket
 
 # hadolint ignore=DL3018
-RUN apk add --no-cache jq git vim rsync unzip bluez skopeo openssh-client
+RUN apk add --no-cache nodejs jq git vim rsync unzip bluez skopeo openssh-client
```

Comment:
This limits us to the node version in the alpine 3.12 repos, right? Unless we override the repo URL.

Conveniently that appears to be `12.22.6-r0` but if we ever bump alpine we will need to be more clear which version of nodejs we are installing and from where.